downstream/assemblies/hub/assembly-syncing-to-cloud-repo.adoc

```diff
@@ -1,36 +1,33 @@
 :_mod-docs-content-type: ASSEMBLY
 [id="assembly-creating-tokens-in-automation-hub"]
-= Configuring {HubNameMain} remote repositories to synchronize content
+= Configuring remote repositories for content syncing
 
 :context: cloud-sync
 
 [role="_abstract"]
 Use remote configurations to configure your {PrivateHubName} to synchronize with {CertifiedName} hosted on `{Console}` or with your collections in {Galaxy}.
 
-Each remote configuration located in {MenuACAdminRemotes} provides information for both the *community* and *rh-certified* repository about when the repository was *last updated*.
-You can add new content to {HubNameMain} at any time using the *Edit* and *Sync* features included on the {MenuACAdminRepositories} page.
+Each remote configuration in {MenuACAdminRemotes} provides information for both the *community* and *rh-certified* repository about when the repository was *last updated*.
+You can add new content to {HubNameMain} at any time by using the *Edit* and *Sync* features included on the {MenuACAdminRepositories} page.
 
-*What's the difference between {Galaxy} and {HubNameMain}?*
+*Understanding the difference between {Galaxy} and {HubNameMain} collections*
 
 Collections published to {Galaxy} are the latest content published by the Ansible community and have no joint support claims associated with them. 
-{Galaxy} is the recommended frontend directory for the Ansible community to access content.
+{Galaxy} is the recommended front end directory for the Ansible community to access content.
 
 Collections published to {HubNameMain} are targeted to joint customers of Red Hat and selected partners. 
 Customers need an Ansible subscription to access and download collections on {HubNameMain}. 
-A certified collection means that Red Hat and partners have a strategic relationship in place and are ready to support joint customers, and that the collections may have had additional testing and validation done against them.
+A certified collection means that Red Hat and partners have a strategic relationship in place and are ready to support joint customers. Collections may also have had additional testing and validation done against them.
```

Comment:
This is a bit of a nit-picky thing, but because it was pointed out to me a couple times in previous peer reviews, it stands out to me. I'm referring to IBM style guide and the use of "may vs. might" [here](https://www.ibm.com/docs/en/ibm-style?topic=word-usage): 

**may** verb
Use only to indicate permission. Otherwise, use _**can**_ to indicate ability, or use _**might**_ to indicate possibility.

Suggestion: replace "may" with "might"